### ISSUE TITLE
Git related error message when importing parcels 

### ISSUE THREAD

**Issue by VeckoTheGecko** [OceanParcels/parcels#1250]
Platform: Windows 10 (WSL)

I've been encountering this error message when running `import parcels` for the first time in a session:
```bash
>>> import parcels
fatal: not a git repository (or any of the parent directories): .git
```

It doesn't actually affect anything, and parcels functions as normal. Its just a bit...weird. I also don't know when I first started encountering it.

Is this a quirk that is known?
I'm happy to look into this issue more, was just getting a feel for whether this is a widespread problem.

@mheen feel free to include any other info here 😄

**Comment by erikvansebille** [comment]
Hmm, I've seen these warnings pop up too. They may be related to the way we determine the version of Parcels? See https://github.com/OceanParcels/parcels/blob/master/parcels/_version.py#L4

Can you check if the same warning occurs when you do
```python
from parcels._version import version
```

**Comment by VeckoTheGecko** [comment]
> Can you check if the same warning occurs when you do
> 
> ```python
> from parcels._version import version
> ```

Yep, same warning occurs when I do that (`version='2.3.2'` by the way)

**Comment by erikvansebille** [comment]
Hi @VeckoTheGecko, I have just released v2.4.0 of parcels via conda; could you check if this issue indeed disappears with the latest version on your system?

**Comment by VeckoTheGecko** [comment]
Yes, that fixed it 🚀🚀
